src/main/java/frc/robot/subsystems/Deployer.java

```diff
@@ -0,0 +1,115 @@
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+///This class is meant to manage the motor that "deploys" the intake, by rotating in order to raise and lower the intake area.
+public class Deployer extends SubsystemBase{
+    private WPI_TalonSRX deployerMotor;
+
+    // The following line references the "protectionMechanism" subsystem, which does not currently exist in this year's code, so I am leaving it commented out. For more detail, see the last comment on this file - currently line 82, as I am writing this
+    /* private ProtectionMechanism protectionMechanism; */
+    
+
+    public Deployer() {
+        int TIMEOUT=100;
+
+        deployerMotor = new WPI_TalonSRX(Constants.DEPLOYER_MOTOR_ID);
+        deployerMotor.configNominalOutputForward(0, TIMEOUT);
+        deployerMotor.configNominalOutputReverse(0, TIMEOUT);
+        deployerMotor.configPeakOutputForward(1, TIMEOUT);
+        deployerMotor.configPeakOutputReverse(-1, TIMEOUT);
+        deployerMotor.setNeutralMode(NeutralMode.Brake);
+        deployerMotor.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder, 0, TIMEOUT);
+        deployerMotor.configForwardLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen);
+        deployerMotor.configReverseLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen);
+        deployerMotor.setSelectedSensorPosition(0);
+
+        //In last years code, there was this following code here involved in "Diagnostics". I'm not sure if it's relevant to what we are doing this year, so I'm leaving it commented out for now
+        /*
+        Robot.getDiagnostics().addDiagnosable(new DiagTalonSrxEncoder("Extender", "Encoder", Constants.DIAG_TALONSRX_ROT, extenderMotor));
+        Robot.getDiagnostics().addDiagnosable(new DiagTalonSrxSwitch("Extender", "Extended Switch", extenderMotor, frc.robot.utils.diag.DiagTalonSrxSwitch.Direction.FORWARD));
+        Robot.getDiagnostics().addDiagnosable(new DiagTalonSrxSwitch("Extender", "Retracted Switch", extenderMotor, frc.robot.utils.diag.DiagTalonSrxSwitch.Direction.REVERSE));
+        */
+    }
+
+    public void resetEncoder() {
+        deployerMotor.setSelectedSensorPosition(0);
+    }
+    
+    // The following function references another function which in turn references the "protectionMechanism" subsystem, which does not currently exist in this year's code, so I am leaving it commented out. For more detail, see the last comment on this file - currently line 82, as I am writing this
+    /*
+    public void move(double speed) {
+        deployerMotor.set(validateExtenderVolt(speed));
+    }
+    */
+
+    public void stop() {
+        deployerMotor.set(0);
+    }
+
+    public double getEncoder() {
+        return deployerMotor.getSelectedSensorPosition();
+    }
+
+    public boolean fwdLimitReached() {
```

Comment:
You have duplicate functions, "fwdLimitReached()" and isDeployerFowardLimitSwitchClosed() are doing the same thing, they're givine limit switch values. Pick one and delete the other duplicate function.